src/index.html

```diff
@@ -3,9 +3,32 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>webpack demo</title>
+    <title>Leaderboard</title>
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+    <header>
+        <h1>Leaderboard</h1>
+    </header>
+    <main>
+        <section id="recent-scores-section">
+            <div id="recent-scores-header-section">
+                <h2>Recent scores</h2>
+                <button id="refresh-button">Refresh</button>
+            </div>
+            <ul id="score-list">
+                <li>Name: 100</li>
+                <li>Name: 20</li>
+                <li>Name: 70</li>
+            </ul>
+        </section>
+        <section id="add-score-section">
+            <h2>Add your score</h2>
+            <form id="score-form">
+                <input type="text" name="name" id="name" placeholder="Your name"><br><br>
+                <input type="number" name="score" id="score" placeholder="Your score"><br><br>
+                <input type="submit" value="Submit">
+            </form>
+        </section>
+    </main>
 </body>
```

Comment:
- [ ] Its required to make the wireframe similar to the provided design. Kindly address this issue.

- Your design
![image](https://user-images.githubusercontent.com/81932478/173933428-0dd768aa-49bc-42ea-a9e6-30779c022e5e.png)

- Provided design
![image](https://user-images.githubusercontent.com/81932478/173933517-034ec38c-c25b-4e7d-8d86-2c2c49b8b396.png)